tests/testthat/test-expect-condition.R

```diff
@@ -153,6 +153,18 @@ test_that("cnd expectations consistently return condition (#1371)", {
   expect_equal(expect_error(f("return value", NULL), regexp = NA), "return value")
 })
 
+test_that("cli width wrapping doesn't affect text matching", {
+  skip_if_not_installed("cli", "3.0.2")
+  skip_if_not_installed("rlang", "1.0.0")
+
+  local_use_cli()
+
+  expect_error(
+    abort("foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz foobarbaz"),
```

Comment:
Is there a reason to not call `cli_abort()` directly?

Reply:
`cli_abort()` will need to be updated after rlang 1.0 is out so I thought I'd leave that out of the equation for now.

Reply:
Ok, makes sense